global.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-linux`
Get-Member.Should be able to be called on IntPtr
```powershell
Expected exactly 2, but got 4.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Member.Tests.ps1: line 46
46:         $results.Count | Should -BeExactly 2
```